docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -207,6 +207,8 @@ All of the above needs to be run only the first time you set up the local dev en
 npm ci
 ```
 
+**NB:** If your ISP has certain network restrictions, the above step (2) may fail without warning. In such a case, you will need to install a VPN or proxy to get this step to pass.
```

Comment:
```suggestion
If you get errors while installing the dependencies, please make sure that you are not in a restricted network or your firewall settings do not prevent you from accessing resources. Some solutions would be use to VPN services if possible and allowed in your environment.
```

Hi @Jo-IE 

Thanks for the PR. I think we should move this change to the troubleshooting area. Second we should update the language to be more neutral.